source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,68 @@
+.phone {
+  $medium-screen: em(640) !default;
+  $large-screen: em(860) !default;
+  $ratio-small: 1;
+  $ratio-large: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(black, 0.8);
+  $phone-padding: 0.4em;
+  $phone-screen-width: 8.5em;
+  $phone-screen-height: 15em;
+  $phone-background: #F8F8F8;
+
+  background: $phone-background;
+  border-radius: (1.2em * $ratio-small);
+  box-shadow: 0 0 1px transparentize(black, 0.4), inset 0 3px 0 white;
+  padding: $phone-padding * $ratio-small;
+  width: $phone-screen-width * $ratio-small + (2 * $phone-padding * $ratio-small);
+
+  @include media($large-screen) {
+    border-radius: (1.2em * $ratio-large);
+    padding: $phone-padding * $ratio-large;
+    width: $phone-screen-width * $ratio-large + (2 * $phone-padding * $ratio-large);
+  }
+
+  .phone-speaker {
+    background: #ABABAB;
+    border-radius: 1em * $ratio-small;
+    box-shadow: 0 2px 0 white, inset 0 0 1px transparentize(black, 0.6);
+    height: (0.3em * $ratio-small);
+    margin: (1em * $ratio-small) auto;    
+    width: $phone-screen-width * $ratio-small * 0.2;
+
+    @include media($large-screen) {
+      height: (0.3em * $ratio-large);
+      margin: (1em * $ratio-large) auto;    
+      width: $phone-screen-width * $ratio-large * 0.2;
+    }    
+  }
+
+  .phone-screen {
+    background: url('https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/cosmin_capitanu_screen.jpg');
+    background-size: cover;
+    border-radius: 0.1em * $ratio-small;
+    box-shadow: 0 3px 0 white, inset 0 0 1px transparentize(black, 0.6);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.